web/src/components/carousel/component/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+import { IoIosArrowDropleftCircle, IoIosArrowDroprightCircle } from 'react-icons/io';
+import './style.css';
+
+interface Props {
+    images: Required<{ image: string }[]>;
+    interval?: number;
+}
+
+const Carousel = ({ images, interval }: Props) => {
+    const [count, setCount] = useState(0);
+    const length = images.length;
+
+    const next = () => {
+        setCount(count === length - 1 ? 0 : count + 1);
+    };
+
+    const previous = () => {
+        setCount(count === 0 ? length - 1 : count - 1);
+    };
+
+    useEffect(() => {
+        const setImage = setInterval(next, interval ? interval : 5000);
+        return () => clearInterval(setImage);
+    });
+
+    return (
+        <div>
+            <IoIosArrowDropleftCircle className="leftArrow" onClick={previous} />
+            {images.map((slide: any, index: any) => {

```

Comment:
Don't necessary has be image. It should be custom component. So you should take Children from props and map them here. Also please remove example images